### DISCUSSION TITLE
How to cite "SS_Fitting Manual" and "SS_Wave_Excitation_Fitting Manual"?

### DISCUSSION THREAD

**Discussion by Melancho7ee** [OpenFAST/openfast#1795] [General]
Dear Jonkman,
        I am recently writing a paper and decide to submit. However, I am confused how to cite the following two manual book:
1. SS_Fitting Theory and User Manual (Tiago Duarte Original: 11/1/2012 V5_18-09-2013)
2. SS_Wave_Excitation_Fitting_Report_draft Theory and User Manual (Alan Wright Jason Jonkman Amy Robertson Greg Hayman Draft: 02/3/2019)

**Comment by jjonkman** [comment]
Dear @Melancho7ee,

Neither of these two documents were every officially published.  You could reference them as unpublished reports.  Alternatively, you could reference the papers that were written when these SS_Fitting solvers were first developed, i.e.: https://www.nrel.gov/docs/fy13osti/58099.pdf for SS_Fitting and https://www.nrel.gov/docs/fy19osti/71865.pdf for SS_Excitation_Fitting

Alternatively, you could reference the websites were the software and documentation can be found.  For SS_Fitting, this is:  https://www.nrel.gov/wind/nwtc/ss-fitting.html.  A similar website doesn't exist for SS_Excitation_Fitting, but you could instead reference the website where SS_Excitation_Fitting is discussed, i.e., the following OpenFAST issue: https://github.com/OpenFAST/openfast/issues/397.

Best regards,
